chainer/training/triggers/interval.py

```diff
@@ -23,7 +23,6 @@ def __init__(self, period, unit):
         self.period = period
         assert unit == 'epoch' or unit == 'iteration'
         self.unit = unit
-        self.count = 0
```

Comment:
Although `count` is not a documented attribute of `IntervalTrigger`, some users could use this attribute. If it is not heavy, could you restore this attribute with a comment that it is for keeping backward compatibility.